packages/opentelemetry-instrumentation-fetch/test/fetch.test.ts

```diff
@@ -613,6 +610,17 @@ describe('fetch', () => {
         'parent span is not root span'
       );
     });
+
+    it('should apply custom attributes', () => {
```

Comment:
I think this test is in wrong `describe` block, you should probably create a new `describe` block because the custom attributes callback should be called everytime not only `when request is NOT successful (wrong url)` ?

Reply:
I've put all the changes in tests in a new describe block now, let me know if any additional cases should be covered